src/backend/src/modules/configuration/configuration.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller, Get } from '@nestjs/common';
+import { ApiTags, ApiOkResponse } from '@nestjs/swagger';
+import { ConfigurationService } from './configuration.service';
+
+@Controller('configuration')
+@ApiTags('configuration')
+export class ConfigurationController {
+  constructor(private configurationService: ConfigurationService) {}
+
+  @Get('/')
+  @ApiOkResponse({
+    description: 'Get feature flag configuration json file',
+  })
+  featureFlags() {
+    return this.configurationService.getFeatures();
```

Comment:
Hey, please rename this method to getConfig
OR if the idea is to send only flags please make appropriate changes in the service to send just the json.feature_flags, and change the endpoint to /flags.

I prefer the first one.

Reply:
i agree, change to getConfig as func name